charts/gardener-extension-shoot-oidc-service/templates/poddisruptionbudget.yaml

```
@@ -4,7 +4,11 @@
 
 ---
 {{- if gt (int .Values.replicaCount) 1 }}
+{{- if semverCompare ">= 1.21-0" .Capabilities.KubeVersion.GitVersion -}}
+apiVersion: policy/v1
+{{- else -}}
 apiVersion: policy/v1beta1
+{{- end -}}
```

```suggestion
{{- if semverCompare ">= 1.21-0" .Capabilities.KubeVersion.GitVersion }}
apiVersion: policy/v1
{{- else }}
apiVersion: policy/v1beta1
{{- end }}
```
See https://helm.sh/docs/chart_template_guide/control_structures/#controlling-whitespace and https://github.com/gardener/gardener-extension-shoot-networking-problemdetector/pull/30